MSAL/test/unit/native_auth/mock/SignInTestsValidatorHelpers.swift

```diff
@@ -32,8 +32,7 @@ class SignInPasswordStartTestsValidatorHelper: SignInPasswordStartDelegateSpy {
             expectation.fulfill()
             return XCTFail("input should be .error")
         }
-
-        self.expectedError = error
```

Comment:
We actually had a problem in the tests that use this function, as they were passing but were not validating properly